### PR TITLE
Updates all mentions of 'master' in documentation to 'main'

### LINKED ISSUE #56: Update mentions of 'master' to 'main'
Problem description:
At time of writing branch 'master' is mentioned in `CONTRIBUTING.md`, `README.md` and `other/MAINTAINING.md`, however this repo uses 'main' instead. Additionally any new forks of repo will default to 'main'.

Suggested solution:
Update the md files mentioned above to say 'main' instead of 'master'.

### PR DESCRIPTION
**Pull request by fildon**
**What**: Update all mentions of 'master' in documentation to 'main'. Resolves https://github.com/kentcdodds/kcd-discord-bot/issues/56

**Why**: This repo uses 'main' as the primary branch. The documentation should reflect that. 

**How**: Just a find and replace 😅 

**Checklist**:
- [x] Documentation
- [N/A] Tests
- [x] Ready to be merged

This is my first ever open source PR ❤️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,6 @@ To get a list of commands, run: `?help` in
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Installation](#installation)
```
Comment (fildon) [inline README.md:35]: I don't believe I changed this line 🤔 
Is it possible doctoc removed it?

Comment (MichaelDeBoey) [reply README.md:35]: It's `doctoc` that removed this line indeed

**Review by MichaelDeBoey — APPROVED** [review]
LGTM! 👊

**Comment by kentcdodds** [comment]
@All-contributors please add @fildon for docs

**Comment by kentcdodds** [comment]
@All-contributors please add @MichaelDeBoey for review

**Pull request merged**